### PR TITLE
BULK VALIDATION description-missing

### PR DESCRIPTION
**Pull request by mSarts**
## Summary

Fixes 30 issues across 30 files.

Fixes no issue numbers.

### CODE REVIEW

File: dotnet-desktop-guide/framework/wpf/app-development/how-to-detect-whether-the-wpf-plug-in-for-firefox-is-installed.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Detect Whether WPF Plug-In for Firefox Is Installed
+description: Learn how to detect whether to WPF plug-in for Firefox is installed, by means of this code sample in HTML.
```
Comment (annmariehitchcock) [inline dotnet-desktop-guide/framework/wpf/app-development/how-to-detect-whether-the-wpf-plug-in-for-firefox-is-installed.md:3]: I think "to" was meant to be "the"?

Comment (mSarts) [reply dotnet-desktop-guide/framework/wpf/app-development/how-to-detect-whether-the-wpf-plug-in-for-firefox-is-installed.md]: Addressed.

**Comment by mSarts** [comment]
**Global effort to fix validation errors**

 @adegeo 

The Content & Learning team is fixing build validation errors on docs.microsoft.com. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. I’d very much appreciate your timely review and feedback on these changes I’ve suggested. Thanks!

**Comment by mSarts** [comment]
@adegeo - I'm following up on this PR. Have you had a chance to review? cc: @v-emilypr

File: dotnet-desktop-guide/framework/wpf/app-development/how-to-refresh-a-page.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "How to: Refresh a Page"
+description: Learn how to refresh a page in this article, using the Refresh%2A method, by means of code examples in C# and Visual Basic.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/app-development/how-to-refresh-a-page.md:3]:
```suggestion
description: Learn how to refresh a page in this article, using the Refresh method, by means of code examples in C# and Visual Basic.
```

**Review by adegeo — APPROVED** [review]

**Pull request merged**